modules/ocf/files/puppet-trigger

```diff
@@ -60,12 +34,12 @@ def switch_to_environment(env):
 
     # TODO: use subprocess.run on python3.5; we don't actually care about stdout here
     subprocess.check_output(
-        (
+        [
             'kinit',
             '-t', '/etc/krb5.keytab',
             'host/{}.ocf.berkeley.edu@OCF.BERKELEY.EDU'.format(hostname),
             'ldapmodify',
-        ),
+        ],
```

Comment:
Actually, the way we build the list of arguments in this script now, it does make a little more sense for it to be a list...